src/frame.c

```diff
@@ -148,19 +148,6 @@ check_window_system (struct frame *f)
 	 : "Window system is not in use or not initialized");
 }
 
-/* Return the value of frame parameter PROP in frame FRAME.  */
-
-Lisp_Object
-get_frame_param (register struct frame *frame, Lisp_Object prop)
-{
-  register Lisp_Object tem;
-
-  tem = Fassq (prop, frame->param_alist);
-  if (EQ (tem, Qnil))
-    return tem;
-  return Fcdr (tem);
```

Comment:
After reading this code again, I wonder if the following is an equivalent function:

```
match assq(prop, frame.param_alist).as_cons() {
    Some(cons) => cons.cdr(),
    None => Qnil,
}
```